flent/tests/http-1up.conf

```diff
@@ -16,6 +16,19 @@ PLOTS['totals'] = {'description': 'TCP bandwidth with delay and jitter',
                    'type': 'meta',
                    'subplots': ('bandwidth', 'http_delay')}
 PLOTS['http_delay_cdf']['cutoff'] = (DELAY,-DELAY)
+
+PLOTS['box_total'] = {'description': 'Candlestick plot of HTTP latency and TCP bandwidth',
+                        'type': 'box',
+                        'axis_labels': ['Mean bandwidth (MBits/s)', 'Mean download time (ms)'],
```

Comment:
They are not mean values when it's not a combination plot (they are samples). Let's just drop the 'axis_labels' parameter completely here.